portfolio/src/main/webapp/script.js

```diff
@@ -14,15 +14,38 @@
 
 /**
  * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+*/
+
+const KEY = "&key=AIzaSyAAHC0kzUB8IDwJlG0DaP2lLyc_haNkNWs"
+const URL = "https://maps.googleapis.com/maps/api/geocode/json?latlng="
+
+async function generateQuery(longitude,latitude){ // function wont have parameters this is just for offline testing.
```

Comment:
Maybe make the argument be an object with `lat` and `lng` keys. This is how they are stored in the Google Maps Javascript SDK([documentation](https://developers.google.com/maps/documentation/javascript/reference/coordinates)) so this will make the function easier to use.